### ISSUE TITLE
Change of key bindings for menu navigation

### ISSUE THREAD

**Issue by crslegend** [alexguirre/RAGENativeUI#128]
For version 1.6.3, i was able to navigate through the menu using the mobile phone navigation keys. However, i am only able to navigate using arrow keys from version 1.7 onwards. Is this expected and is there any way for me to change the key binding?

**Comment by xXFireBunnyXx** [comment]
i would love for a way to change the keybinds as well because currently its a problem for me

**Comment by alexguirre** [comment]
In 1.7 the default controls changed from `CellphoneUp/Down/Left/Right` to `FrontendUp/Down/Left/Right` which are the controls used by the original game menus. I didn't notice that these frontend controls couldn't be rebinded in the game settings, but the cellphones ones could, I guess that's the issue? Because by default both are binded to the arrow keys.

At the moment, the only way to change the defaults is to recompile with different GameControls here:
https://github.com/alexguirre/RAGENativeUI/blob/48e10749d6fc9357ec531017b6a1ac4ee8c8aea2/Source/UIMenu.cs#L333-L347

**Comment by crslegend** [comment]
hi @alexguirre! thanks for your reply. so meaning there is no way currently for me to rebind the keys solely from GTA V menu or through any config files? cus im not really sure how to go about recompiling the gamecontrols with the code quoted

**Comment by alexguirre** [comment]
Yeah, recompiling is the only way for now. Will probably add an .ini file in the next version to allow to change the default game controls.

To recompile you need Visual Studio 2019 with .NET Framework 4.7.2, download this repository, modify the `SetKey` calls quoted with the cellphone [game controls](https://docs.ragepluginhook.net/?topic=html/T_Rage_GameControl.htm) and build it. It should create a new RAGENativeUI.dll file.

These are the game controls used before 1.7:
https://github.com/alexguirre/RAGENativeUI/blob/c5681d6ee946e96e0a293f6bb55abc7cfb1104c8/Source/UIMenu.cs#L261-L272

**Comment by crslegend** [comment]
Hi @alexguirre, I tried to modify and compile, but I am getting compilation errors at these few lines

![Screenshot 2021-07-18 113728](https://user-images.githubusercontent.com/27951537/126054866-f752af65-d21f-4ae1-a812-503eea53d81f.jpg)

![image](https://user-images.githubusercontent.com/27951537/126054877-f570736b-643c-4aec-b058-a8d418c22e04.png)

The errors seems to be coming from lines 271 and 272. Am I doing it correctly?

**Comment by alexguirre** [comment]
Forgot to say it needs C# 9, try updating Visual Studio to the latest version

**Comment by crslegend** [comment]
Its working now! thanks alot!
